episodes/12-fire-forget-tasks.md

```diff
@@ -0,0 +1,392 @@
+---
+title: "Fire-and-forget tasks"
+teaching: 60
+exercises: 30
+---
+
+:::::::::::::::::::::::::::::::::::::: questions
+- "How do execute work in parallel?"
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+- "First objective."
```

Comment:
Fix this.

Reply:
Added two objectives. Resolved through a commit. Will pay attention to the objectives in the followup chapters.